internal/i18n/i18n_test.go

```diff
@@ -55,6 +55,10 @@ func TestI18n_matching(t *testing.T) {
 				po.ParseFile(pth)
 				pos = append(pos, po)
 
+				if po.Language == "" {
+					t.Errorf("MISSING LANGUAGE: path: %q", pth)
```

Comment:
if there is some kind of formatting error lower in the file, it can apparently cause the language to be missing. Being able to quickly identify the offending file is necessary and quite helpful.